lib/plugins/aws/package/compile/events/sqs/index.test.js

```diff
@@ -618,5 +626,15 @@ describe('AwsCompileSQSEvents #2', () => {
       const aliasLogicalId = naming.getLambdaProvisionedConcurrencyAliasLogicalId('foo');
       expect(eventSourceMappingResource.DependsOn).to.include(aliasLogicalId);
     });
+
+    it('should have correct batch size', () => {
+      expect(JSON.stringify(eventSourceMappingResource.Properties.BatchSize)).to.equal('10');
```

Comment:
Let's not use `JSON.stringify`. Simply confirm literal values

Reply:
Agreed and updated